x-pack/plugins/security_solution/public/management/pages/policy/view/policy_forms/locked_card.tsx

```diff
@@ -42,40 +42,40 @@ export const LockedPolicyCard = memo(() => {
             </strong>
           </h3>
         }
-        description={
-          <EuiFlexGroup className="lockedCardDescription" direction="column" gutterSize="none">
-            <EuiFlexItem>
-              <h4>
-                <EuiTextColor color="subdued">
-                  <FormattedMessage
-                    id="xpack.securitySolution.endpoint.policy.details.upgradeToPlatinum"
-                    defaultMessage="Upgrade to Elastic Platinum"
-                  />
-                </EuiTextColor>
-              </h4>
-            </EuiFlexItem>
-            <EuiFlexItem>
-              <p>
+        description={false}
```

Comment:
@parkiino please review this change. The docs for `EuiCard` say "In the event that you need more than just paragraph text for the description, you can pass anything you need as the children of the component.". Everything in `description` gets nested with a `<p>`, so previously we were getting this error.

```
  console.error
      Warning: validateDOMNesting(...): <h4> cannot appear as a descendant of <p>.
          in h4
          in p (created by EuiCard)
 ```
 I noticed the error introduced in the jest tests. Please double check these tests for errors before merging. Even though the tests pass, we want to keep this console clear. 

I have been struggling for over an hour to get to this page, but I can't seem to get the data right. Could you please confirm that this does not change anything visually in the front end? Thanks!!

Reply:
If we put what was previously in the description in the children, we would just need to add an `<EuiText>` component around the `<EuiFlexItem />` to get the UI looking like it was before.

```suggestion
<EuiCard>
        <EuiFlexGroup className="lockedCardDescription" direction="column" gutterSize="none">
             <EuiText>
                <EuiFlexItem> ... </EuiFlexItem>
                <EuiFlexItem> ... </EuiFlexItem>
             </EuiText>
         </EuiFlexGroup>
     </EuiCard>
```